### ISSUE TITLE
./buildall fails

### ISSUE THREAD

**Issue by ngtuna** [bitnami/minideb#29]
I ran `./buildall` and it failed. I have `debootstrap`, `jq` and `debian-archive-keyring` installed already. The detail log is below:

```
$ sudo ./buildall
============================================
Building bitnami/minideb:jessie
============================================
Building base in /tmp/tmp.8pNqJo6ose
I: Retrieving InRelease 
I: Failed to retrieve InRelease
I: Retrieving Release 
I: Retrieving Release.gpg 
I: Checking Release signature
I: Valid Release signature (key id 75DDC3C4A499F1A18CB5F3C8CBF8D6FD518E17E1)
I: Retrieving Packages 
I: Validating Packages 
I: Resolving dependencies of required packages...
I: Resolving dependencies of base packages...
I: Found additional required dependencies: debconf debianutils gcc-4.9-base gpgv initscripts insserv libacl1 libattr1 libaudit-common libaudit1 libblkid1 libbz2-1.0 libc6 libdb5.3 libdebconfclient0 libgcc1 libgcrypt20 libgpg-error0 liblzma5 libmount1 libncurses5 libpam-modules libpam-modules-bin libpam-runtime libpam0g libpcre3 libselinux1 libsemanage-common libsemanage1 libsepol1 libslang2 libsmartcols1 libsystemd0 libtinfo5 libustr-1.0-1 libuuid1 multiarch-support perl-base sensible-utils startpar sysvinit-utils zlib1g 
I: Found additional base dependencies: gnupg libapt-pkg4.12 libreadline6 libstdc++6 libusb-0.1-4 readline-common 
I: Checking component main on http://httpredir.debian.org/debian...
I: Retrieving libacl1 2.2.52-2
I: Validating libacl1 2.2.52-2
I: Retrieving adduser 3.113+nmu3
I: Validating adduser 3.113+nmu3
I: Retrieving apt 1.0.9.8.4
I: Validating apt 1.0.9.8.4
I: Retrieving libapt-pkg4.12 1.0.9.8.4
I: Validating libapt-pkg4.12 1.0.9.8.4
I: Retrieving libattr1 1:2.4.47-2
I: Validating libattr1 1:2.4.47-2
I: Retrieving libaudit-common 1:2.4-1
I: Validating libaudit-common 1:2.4-1
I: Retrieving libaudit1 1:2.4-1+b1
I: Validating libaudit1 1:2.4-1+b1
I: Retrieving base-files 8+deb8u8
I: Validating base-files 8+deb8u8
I: Retrieving base-passwd 3.5.37
I: Validating base-passwd 3.5.37
I: Retrieving bash 4.3-11+deb8u1
I: Validating bash 4.3-11+deb8u1
I: Retrieving libbz2-1.0 1.0.6-7+b3
I: Validating libbz2-1.0 1.0.6-7+b3
I: Retrieving libdebconfclient0 0.192
I: Validating libdebconfclient0 0.192
I: Retrieving coreutils 8.23-4
I: Validating coreutils 8.23-4
I: Retrieving dash 0.5.7-4+b1
I: Validating dash 0.5.7-4+b1
I: Retrieving libdb5.3 5.3.28-9
I: Validating libdb5.3 5.3.28-9
I: Retrieving debconf 1.5.56
I: Validating debconf 1.5.56
I: Retrieving debian-archive-keyring 2014.3
I: Validating debian-archive-keyring 2014.3
I: Retrieving debianutils 4.4+b1
I: Validating debianutils 4.4+b1
I: Retrieving diffutils 1:3.3-1+b1
I: Validating diffutils 1:3.3-1+b1
I: Retrieving dpkg 1.17.27
I: Validating dpkg 1.17.27
I: Retrieving findutils 4.4.2-9+b1
I: Validating findutils 4.4.2-9+b1
I: Retrieving gcc-4.9-base 4.9.2-10
I: Validating gcc-4.9-base 4.9.2-10
I: Retrieving libgcc1 1:4.9.2-10
I: Validating libgcc1 1:4.9.2-10
I: Retrieving libstdc++6 4.9.2-10
I: Validating libstdc++6 4.9.2-10
I: Retrieving libc-bin 2.19-18+deb8u9
I: Validating libc-bin 2.19-18+deb8u9
I: Retrieving libc6 2.19-18+deb8u9
I: Validating libc6 2.19-18+deb8u9
I: Retrieving multiarch-support 2.19-18+deb8u9
I: Validating multiarch-support 2.19-18+deb8u9
I: Retrieving gnupg 1.4.18-7+deb8u3
I: Validating gnupg 1.4.18-7+deb8u3
I: Retrieving gpgv 1.4.18-7+deb8u3
I: Validating gpgv 1.4.18-7+deb8u3
I: Retrieving grep 2.20-4.1
I: Validating grep 2.20-4.1
I: Retrieving gzip 1.6-4
I: Validating gzip 1.6-4
I: Retrieving hostname 3.15
I: Validating hostname 3.15
I: Retrieving insserv 1.14.0-5
I: Validating insserv 1.14.0-5
I: Retrieving libgcrypt20 1.6.3-2+deb8u2
I: Validating libgcrypt20 1.6.3-2+deb8u2
I: Retrieving libgpg-error0 1.17-3
I: Validating libgpg-error0 1.17-3
I: Retrieving libselinux1 2.3-2
I: Validating libselinux1 2.3-2
I: Retrieving libsemanage-common 2.3-1
I: Validating libsemanage-common 2.3-1
I: Retrieving libsemanage1 2.3-1+b1
I: Validating libsemanage1 2.3-1+b1
I: Retrieving libsepol1 2.3-2
I: Validating libsepol1 2.3-2
I: Retrieving libusb-0.1-4 2:0.1.12-25
I: Validating libusb-0.1-4 2:0.1.12-25
I: Retrieving lsb-base 4.1+Debian13+nmu1
I: Validating lsb-base 4.1+Debian13+nmu1
I: Retrieving mawk 1.3.3-17
I: Validating mawk 1.3.3-17
I: Retrieving libncurses5 5.9+20140913-1+b1
I: Validating libncurses5 5.9+20140913-1+b1
I: Retrieving libtinfo5 5.9+20140913-1+b1
I: Validating libtinfo5 5.9+20140913-1+b1
I: Retrieving ncurses-base 5.9+20140913-1
I: Validating ncurses-base 5.9+20140913-1
I: Retrieving libpam-modules 1.1.8-3.1+deb8u2
I: Validating libpam-modules 1.1.8-3.1+deb8u2
I: Retrieving libpam-modules-bin 1.1.8-3.1+deb8u2
I: Validating libpam-modules-bin 1.1.8-3.1+deb8u2
I: Retrieving libpam-runtime 1.1.8-3.1+deb8u2
I: Validating libpam-runtime 1.1.8-3.1+deb8u2
I: Retrieving libpam0g 1.1.8-3.1+deb8u2
I: Validating libpam0g 1.1.8-3.1+deb8u2
I: Retrieving libpcre3 2:8.35-3.3+deb8u4
I: Validating libpcre3 2:8.35-3.3+deb8u4
I: Retrieving perl-base 5.20.2-3+deb8u6
I: Validating perl-base 5.20.2-3+deb8u6
I: Retrieving libreadline6 6.3-8+b3
I: Validating libreadline6 6.3-8+b3
I: Retrieving readline-common 6.3-8
I: Validating readline-common 6.3-8
I: Retrieving sed 4.2.2-4+deb8u1
I: Validating sed 4.2.2-4+deb8u1
I: Retrieving sensible-utils 0.0.9
I: Validating sensible-utils 0.0.9
I: Retrieving login 1:4.2-3+deb8u3
I: Validating login 1:4.2-3+deb8u3
I: Retrieving passwd 1:4.2-3+deb8u3
I: Validating passwd 1:4.2-3+deb8u3
I: Retrieving libslang2 2.3.0-2
I: Validating libslang2 2.3.0-2
I: Retrieving startpar 0.59-3
I: Validating startpar 0.59-3
I: Retrieving libsystemd0 215-17+deb8u7
I: Validating libsystemd0 215-17+deb8u7
I: Retrieving initscripts 2.88dsf-59
I: Validating initscripts 2.88dsf-59
I: Retrieving sysv-rc 2.88dsf-59
I: Validating sysv-rc 2.88dsf-59
I: Retrieving sysvinit-utils 2.88dsf-59
I: Validating sysvinit-utils 2.88dsf-59
I: Retrieving tar 1.27.1-2+deb8u1
I: Validating tar 1.27.1-2+deb8u1
I: Retrieving tzdata 2017b-0+deb8u1
I: Validating tzdata 2017b-0+deb8u1
I: Retrieving libustr-1.0-1 1.0.4-3+b2
I: Validating libustr-1.0-1 1.0.4-3+b2
I: Retrieving bsdutils 1:2.25.2-6
I: Validating bsdutils 1:2.25.2-6
I: Retrieving libblkid1 2.25.2-6
I: Validating libblkid1 2.25.2-6
I: Retrieving libmount1 2.25.2-6
I: Validating libmount1 2.25.2-6
I: Retrieving libsmartcols1 2.25.2-6
I: Validating libsmartcols1 2.25.2-6
I: Retrieving libuuid1 2.25.2-6
I: Validating libuuid1 2.25.2-6
I: Retrieving mount 2.25.2-6
I: Validating mount 2.25.2-6
I: Retrieving util-linux 2.25.2-6
I: Validating util-linux 2.25.2-6
I: Retrieving liblzma5 5.1.1alpha+20120614-2+b3
I: Validating liblzma5 5.1.1alpha+20120614-2+b3
I: Retrieving zlib1g 1:1.2.8.dfsg-2+b1
I: Validating zlib1g 1:1.2.8.dfsg-2+b1
I: Chosen extractor for .deb packages: dpkg-deb
I: Extracting libacl1...
I: Extracting adduser...
I: Extracting libattr1...
I: Extracting libaudit-common...
I: Extracting libaudit1...
I: Extracting base-files...
I: Extracting base-passwd...
I: Extracting bash...
I: Extracting libbz2-1.0...
I: Extracting libdebconfclient0...
I: Extracting coreutils...
I: Extracting dash...
I: Extracting libdb5.3...
I: Extracting debconf...
I: Extracting debian-archive-keyring...
I: Extracting debianutils...
I: Extracting diffutils...
I: Extracting dpkg...
I: Extracting findutils...
I: Extracting gcc-4.9-base...
I: Extracting libgcc1...
I: Extracting libc-bin...
I: Extracting libc6...
I: Extracting multiarch-support...
I: Extracting gpgv...
I: Extracting grep...
I: Extracting gzip...
I: Extracting hostname...
I: Extracting insserv...
I: Extracting libgcrypt20...
I: Extracting libgpg-error0...
I: Extracting libselinux1...
I: Extracting libsemanage-common...
I: Extracting libsemanage1...
I: Extracting libsepol1...
I: Extracting lsb-base...
I: Extracting mawk...
I: Extracting libncurses5...
I: Extracting libtinfo5...
I: Extracting ncurses-base...
I: Extracting libpam-modules...
I: Extracting libpam-modules-bin...
I: Extracting libpam-runtime...
I: Extracting libpam0g...
I: Extracting libpcre3...
I: Extracting perl-base...
I: Extracting sed...
I: Extracting sensible-utils...
I: Extracting login...
I: Extracting passwd...
I: Extracting libslang2...
I: Extracting startpar...
I: Extracting libsystemd0...
I: Extracting initscripts...
I: Extracting sysv-rc...
I: Extracting sysvinit-utils...
I: Extracting tar...
I: Extracting tzdata...
I: Extracting libustr-1.0-1...
I: Extracting bsdutils...
I: Extracting libblkid1...
I: Extracting libmount1...
I: Extracting libsmartcols1...
I: Extracting libuuid1...
I: Extracting mount...
I: Extracting util-linux...
I: Extracting liblzma5...
I: Extracting zlib1g...
I: Installing core packages...
I: Unpacking required packages...
I: Unpacking libacl1:amd64...
I: Unpacking adduser...
I: Unpacking libattr1:amd64...
I: Unpacking libaudit-common...
I: Unpacking libaudit1:amd64...
I: Unpacking base-files...
I: Unpacking base-passwd...
I: Unpacking bash...
I: Unpacking libbz2-1.0:amd64...
I: Unpacking libdebconfclient0:amd64...
I: Unpacking coreutils...
I: Unpacking dash...
I: Unpacking libdb5.3:amd64...
I: Unpacking debconf...
I: Unpacking debian-archive-keyring...
I: Unpacking debianutils...
I: Unpacking diffutils...
I: Unpacking dpkg...
I: Unpacking findutils...
I: Unpacking gcc-4.9-base:amd64...
I: Unpacking libgcc1:amd64...
I: Unpacking libc-bin...
I: Unpacking libc6:amd64...
I: Unpacking multiarch-support...
I: Unpacking gpgv...
I: Unpacking grep...
I: Unpacking gzip...
I: Unpacking hostname...
I: Unpacking insserv...
I: Unpacking libgcrypt20:amd64...
I: Unpacking libgpg-error0:amd64...
I: Unpacking libselinux1:amd64...
I: Unpacking libsemanage-common...
I: Unpacking libsemanage1:amd64...
I: Unpacking libsepol1:amd64...
I: Unpacking lsb-base...
I: Unpacking mawk...
I: Unpacking libncurses5:amd64...
I: Unpacking libtinfo5:amd64...
I: Unpacking ncurses-base...
I: Unpacking libpam-modules:amd64...
I: Unpacking libpam-modules-bin...
I: Unpacking libpam-runtime...
I: Unpacking libpam0g:amd64...
I: Unpacking libpcre3:amd64...
I: Unpacking perl-base...
I: Unpacking sed...
I: Unpacking sensible-utils...
I: Unpacking login...
I: Unpacking passwd...
I: Unpacking libslang2:amd64...
I: Unpacking startpar...
I: Unpacking libsystemd0:amd64...
I: Unpacking initscripts...
I: Unpacking sysv-rc...
I: Unpacking sysvinit-utils...
I: Unpacking tar...
I: Unpacking tzdata...
I: Unpacking libustr-1.0-1:amd64...
I: Unpacking bsdutils...
I: Unpacking libblkid1:amd64...
I: Unpacking libmount1:amd64...
I: Unpacking libsmartcols1:amd64...
I: Unpacking libuuid1:amd64...
I: Unpacking mount...
I: Unpacking util-linux...
I: Unpacking liblzma5:amd64...
I: Unpacking zlib1g:amd64...
I: Configuring required packages...
I: Configuring lsb-base...
I: Configuring sensible-utils...
I: Configuring ncurses-base...
I: Configuring libsemanage-common...
I: Configuring gcc-4.9-base:amd64...
I: Configuring libaudit-common...
I: Configuring libc6:amd64...
I: Configuring startpar...
I: Configuring libc-bin...
I: Configuring diffutils...
I: Configuring insserv...
I: Configuring findutils...
I: Configuring debianutils...
I: Configuring hostname...
I: Configuring multiarch-support...
I: Configuring mawk...
I: Configuring libpcre3:amd64...
I: Configuring libbz2-1.0:amd64...
I: Configuring libgpg-error0:amd64...
I: Configuring base-files...
I: Configuring libdebconfclient0:amd64...
I: Configuring libselinux1:amd64...
I: Configuring libslang2:amd64...
I: Configuring libsepol1:amd64...
I: Configuring libgcc1:amd64...
I: Configuring libustr-1.0-1:amd64...
I: Configuring libsmartcols1:amd64...
I: Configuring libaudit1:amd64...
I: Configuring libtinfo5:amd64...
I: Configuring libattr1:amd64...
I: Configuring liblzma5:amd64...
I: Configuring base-passwd...
I: Configuring libgcrypt20:amd64...
I: Configuring libdb5.3:amd64...
I: Configuring zlib1g:amd64...
I: Configuring gpgv...
I: Configuring libsystemd0:amd64...
I: Configuring libsemanage1:amd64...
I: Configuring sysvinit-utils...
I: Configuring debian-archive-keyring...
I: Configuring libacl1:amd64...
I: Configuring libncurses5:amd64...
I: Configuring bsdutils...
I: Configuring coreutils...
I: Configuring tar...
I: Configuring dpkg...
I: Configuring sed...
I: Configuring perl-base...
I: Configuring grep...
I: Configuring debconf...
I: Configuring tzdata...
I: Configuring gzip...
I: Configuring dash...
I: Configuring sysv-rc...
I: Configuring libpam0g:amd64...
I: Configuring libpam-modules-bin...
I: Configuring bash...
I: Configuring libpam-modules:amd64...
I: Configuring libpam-runtime...
I: Configuring passwd...
I: Configuring login...
I: Configuring adduser...
I: Configuring libuuid1:amd64...
I: Configuring libblkid1:amd64...
I: Configuring libmount1:amd64...
I: Configuring mount...
I: Configuring initscripts...
I: Configuring util-linux...
I: Configuring libc-bin...
I: Unpacking the base system...
I: Unpacking apt...
I: Unpacking libapt-pkg4.12:amd64...
I: Unpacking libstdc++6:amd64...
I: Unpacking gnupg...
I: Unpacking libusb-0.1-4:amd64...
I: Unpacking libreadline6:amd64...
I: Unpacking readline-common...
I: Configuring the base system...
I: Configuring readline-common...
I: Configuring libstdc++6:amd64...
I: Configuring libapt-pkg4.12:amd64...
I: Configuring libusb-0.1-4:amd64...
I: Configuring libreadline6:amd64...
I: Configuring gnupg...
I: Configuring apt...
I: Configuring libc-bin...
I: Base system installed successfully.
Get:1 http://security.debian.org jessie/updates InRelease [63.1 kB]
Ign http://httpredir.debian.org jessie InRelease
Get:2 http://security.debian.org jessie/updates/main amd64 Packages [409 kB]
Hit http://httpredir.debian.org jessie Release.gpg
Get:3 http://security.debian.org jessie/updates/main Translation-en [215 kB]
Hit http://httpredir.debian.org jessie Release
Get:4 http://httpredir.debian.org jessie/main amd64 Packages [6790 kB]
Get:5 http://httpredir.debian.org jessie/main Translation-en [4583 kB]
Fetched 12.1 MB in 4s (2486 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
The following packages will be upgraded:
  login passwd
2 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Need to get 1693 kB of archives.
After this operation, 2048 B of additional disk space will be used.
Get:1 http://security.debian.org/ jessie/updates/main login amd64 1:4.2-3+deb8u4 [722 kB]
Get:2 http://security.debian.org/ jessie/updates/main passwd amd64 1:4.2-3+deb8u4 [971 kB]
perl: warning: Setting locale failed.
perl: warning: Please check that your locale settings:
	LANGUAGE = (unset),
	LC_ALL = (unset),
	LANG = "en_US.UTF-8"
    are supported and installed on your system.
perl: warning: Falling back to the standard locale ("C").
debconf: delaying package configuration, since apt-utils is not installed
Fetched 1693 kB in 0s (2109 kB/s)
E: Can not write log (Is /dev/pts mounted?) - posix_openpt (2: No such file or directory)
(Reading database ... 6273 files and directories currently installed.)
Preparing to unpack .../login_1%3a4.2-3+deb8u4_amd64.deb ...
Unpacking login (1:4.2-3+deb8u4) over (1:4.2-3+deb8u3) ...
Setting up login (1:4.2-3+deb8u4) ...
(Reading database ... 6273 files and directories currently installed.)
Preparing to unpack .../passwd_1%3a4.2-3+deb8u4_amd64.deb ...
Unpacking passwd (1:4.2-3+deb8u4) over (1:4.2-3+deb8u3) ...
Setting up passwd (1:4.2-3+deb8u4) ...
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name                    Version                  Architecture Description
+++-=======================-========================-============-======================================================================
ii  adduser                 3.113+nmu3               all          add and remove users and groups
ii  apt                     1.0.9.8.4                amd64        commandline package manager
ii  base-files              8+deb8u8                 amd64        Debian base system miscellaneous files
ii  base-passwd             3.5.37                   amd64        Debian base system master password and group files
ii  bash                    4.3-11+deb8u1            amd64        GNU Bourne Again SHell
ii  bsdutils                1:2.25.2-6               amd64        basic utilities from 4.4BSD-Lite
ii  coreutils               8.23-4                   amd64        GNU core utilities
ii  dash                    0.5.7-4+b1               amd64        POSIX-compliant shell
ii  debconf                 1.5.56                   all          Debian configuration management system
ii  debian-archive-keyring  2014.3                   all          GnuPG archive keys of the Debian archive
ii  debianutils             4.4+b1                   amd64        Miscellaneous utilities specific to Debian
ii  diffutils               1:3.3-1+b1               amd64        File comparison utilities
ii  dpkg                    1.17.27                  amd64        Debian package management system
ii  findutils               4.4.2-9+b1               amd64        utilities for finding files--find, xargs
ii  gcc-4.9-base:amd64      4.9.2-10                 amd64        GCC, the GNU Compiler Collection (base package)
ii  gnupg                   1.4.18-7+deb8u3          amd64        GNU privacy guard - a free PGP replacement
ii  gpgv                    1.4.18-7+deb8u3          amd64        GNU privacy guard - signature verification tool
ii  grep                    2.20-4.1                 amd64        GNU grep, egrep and fgrep
ii  gzip                    1.6-4                    amd64        GNU compression utilities
ii  hostname                3.15                     amd64        utility to set/show the host name or domain name
ii  initscripts             2.88dsf-59               amd64        scripts for initializing and shutting down the system
ii  insserv                 1.14.0-5                 amd64        boot sequence organizer using LSB init.d script dependency information
ii  libacl1:amd64           2.2.52-2                 amd64        Access control list shared library
ii  libapt-pkg4.12:amd64    1.0.9.8.4                amd64        package management runtime library
ii  libattr1:amd64          1:2.4.47-2               amd64        Extended attribute shared library
ii  libaudit-common         1:2.4-1                  all          Dynamic library for security auditing - common files
ii  libaudit1:amd64         1:2.4-1+b1               amd64        Dynamic library for security auditing
ii  libblkid1:amd64         2.25.2-6                 amd64        block device id library
ii  libbz2-1.0:amd64        1.0.6-7+b3               amd64        high-quality block-sorting file compressor library - runtime
ii  libc-bin                2.19-18+deb8u9           amd64        GNU C Library: Binaries
ii  libc6:amd64             2.19-18+deb8u9           amd64        GNU C Library: Shared libraries
ii  libdb5.3:amd64          5.3.28-9                 amd64        Berkeley v5.3 Database Libraries [runtime]
ii  libdebconfclient0:amd64 0.192                    amd64        Debian Configuration Management System (C-implementation library)
ii  libgcc1:amd64           1:4.9.2-10               amd64        GCC support library
ii  libgcrypt20:amd64       1.6.3-2+deb8u2           amd64        LGPL Crypto library - runtime library
ii  libgpg-error0:amd64     1.17-3                   amd64        library for common error values and messages in GnuPG components
ii  liblzma5:amd64          5.1.1alpha+20120614-2+b3 amd64        XZ-format compression library
ii  libmount1:amd64         2.25.2-6                 amd64        device mounting library
ii  libncurses5:amd64       5.9+20140913-1+b1        amd64        shared libraries for terminal handling
ii  libpam-modules:amd64    1.1.8-3.1+deb8u2         amd64        Pluggable Authentication Modules for PAM
ii  libpam-modules-bin      1.1.8-3.1+deb8u2         amd64        Pluggable Authentication Modules for PAM - helper binaries
ii  libpam-runtime          1.1.8-3.1+deb8u2         all          Runtime support for the PAM library
ii  libpam0g:amd64          1.1.8-3.1+deb8u2         amd64        Pluggable Authentication Modules library
ii  libpcre3:amd64          2:8.35-3.3+deb8u4        amd64        Perl 5 Compatible Regular Expression Library - runtime files
ii  libreadline6:amd64      6.3-8+b3                 amd64        GNU readline and history libraries, run-time libraries
ii  libselinux1:amd64       2.3-2                    amd64        SELinux runtime shared libraries
ii  libsemanage-common      2.3-1                    all          Common files for SELinux policy management libraries
ii  libsemanage1:amd64      2.3-1+b1                 amd64        SELinux policy management library
ii  libsepol1:amd64         2.3-2                    amd64        SELinux library for manipulating binary security policies
ii  libslang2:amd64         2.3.0-2                  amd64        S-Lang programming library - runtime version
ii  libsmartcols1:amd64     2.25.2-6                 amd64        smart column output alignment library
ii  libstdc++6:amd64        4.9.2-10                 amd64        GNU Standard C++ Library v3
ii  libsystemd0:amd64       215-17+deb8u7            amd64        systemd utility library
ii  libtinfo5:amd64         5.9+20140913-1+b1        amd64        shared low-level terminfo library for terminal handling
ii  libusb-0.1-4:amd64      2:0.1.12-25              amd64        userspace USB programming library
ii  libustr-1.0-1:amd64     1.0.4-3+b2               amd64        Micro string library: shared library
ii  libuuid1:amd64          2.25.2-6                 amd64        Universally Unique ID library
ii  login                   1:4.2-3+deb8u4           amd64        system login tools
ii  lsb-base                4.1+Debian13+nmu1        all          Linux Standard Base 4.1 init script functionality
ii  mawk                    1.3.3-17                 amd64        a pattern scanning and text processing language
ii  mount                   2.25.2-6                 amd64        Tools for mounting and manipulating filesystems
ii  multiarch-support       2.19-18+deb8u9           amd64        Transitional package to ensure multiarch compatibility
ii  ncurses-base            5.9+20140913-1           all          basic terminal type definitions
ii  passwd                  1:4.2-3+deb8u4           amd64        change and administer password and group data
ii  perl-base               5.20.2-3+deb8u6          amd64        minimal Perl system
ii  readline-common         6.3-8                    all          GNU readline and history libraries, common files
ii  sed                     4.2.2-4+deb8u1           amd64        The GNU sed stream editor
ii  sensible-utils          0.0.9                    all          Utilities for sensible alternative selection
ii  startpar                0.59-3                   amd64        run processes in parallel and multiplex their output
ii  sysv-rc                 2.88dsf-59               all          System-V-like runlevel change mechanism
ii  sysvinit-utils          2.88dsf-59               amd64        System-V-like utilities
ii  tar                     1.27.1-2+deb8u1          amd64        GNU version of the tar archiving utility
ii  tzdata                  2017b-0+deb8u1           all          time zone and daylight-saving time data
ii  util-linux              2.25.2-6                 amd64        Miscellaneous system utilities
ii  zlib1g:amd64            1:1.2.8.dfsg-2+b1        amd64        compression library - runtime
Applying docker-specific tweaks
+ echo exit 101 > '/tmp/tmp.8pNqJo6ose/usr/sbin/policy-rc.d'
+ rootfs_chroot dpkg-divert --local --rename --add /sbin/initctl
+ PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+ /usr/sbin/chroot /tmp/tmp.8pNqJo6ose dpkg-divert --local --rename --add /sbin/initctl
Adding 'local diversion of /sbin/initctl to /sbin/initctl.distrib'
+ cp -a /tmp/tmp.8pNqJo6ose/usr/sbin/policy-rc.d /tmp/tmp.8pNqJo6ose/sbin/initctl
+ sed -i 's/^exit.*/exit 0/' /tmp/tmp.8pNqJo6ose/sbin/initctl
+ rootfs_chroot apt-get clean
+ PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+ /usr/sbin/chroot /tmp/tmp.8pNqJo6ose apt-get clean
+ echo force-unsafe-io > '/tmp/tmp.8pNqJo6ose/etc/dpkg/dpkg.cfg.d/docker-apt-speedup'
+ cat > '/tmp/tmp.8pNqJo6ose/etc/apt/apt.conf.d/docker-clean'
+ echo Acquire::Languages 'none' > '/tmp/tmp.8pNqJo6ose/etc/apt/apt.conf.d/docker-no-languages'
+ echo Acquire::GzipIndexes 'true' > '/tmp/tmp.8pNqJo6ose/etc/apt/apt.conf.d/docker-gzip-indexes'
+ echo Apt::AutoRemove::SuggestsImportant 'false' > '/tmp/tmp.8pNqJo6ose/etc/apt/apt.conf.d/docker-autoremove-suggests'
Trimming down
Total size
59M	/tmp/tmp.8pNqJo6ose
Package sizes
adduser 1066
apt 3178
base-files 413
base-passwd 185
bash 5010
bsdutils 181
coreutils 14249
dash 191
debconf 614
debian-archive-keyring 108
debianutils 147
diffutils 950
dpkg 6643
findutils 1406
gcc-4.9-base 218
gnupg 4893
gpgv 414
grep 1272
gzip 239
hostname 59
initscripts 165
insserv 183
libacl1 80
libapt-pkg4.12 2664
libattr1 30
libaudit-common 49
libaudit1 157
libblkid1 326
libbz2-1.0 114
libc-bin 3254
libc6 10229
libdb5.3 1812
libdebconfclient0 53
libgcc1 129
libgcrypt20 1002
libgpg-error0 444
liblzma5 309
libmount1 357
libncurses5 306
libpam-modules 860
libpam-modules-bin 248
libpam-runtime 1447
libpam0g 252
libpcre3 618
libreadline6 419
libselinux1 213
libsemanage-common 65
libsemanage1 245
libsepol1 339
libslang2 1543
libsmartcols1 209
libstdc++6 1319
libsystemd0 183
libtinfo5 480
libusb-0.1-4 42
libustr-1.0-1 287
libuuid1 89
login 2186
lsb-base 72
mawk 198
mount 357
multiarch-support 194
ncurses-base 371
passwd 2137
perl-base 5097
readline-common 109
sed 575
sensible-utils 110
startpar 95
sysv-rc 125
sysvinit-utils 147
tar 2261
tzdata 1710
util-linux 2733
zlib1g 179
Largest dirs
1868	/tmp/tmp.8pNqJo6ose/usr/share/perl/5.20.2/unicore/lib
2608	/tmp/tmp.8pNqJo6ose/sbin
2844	/tmp/tmp.8pNqJo6ose/usr/share/perl/5.20.2/unicore
3248	/tmp/tmp.8pNqJo6ose/usr/share/perl/5.20.2
3252	/tmp/tmp.8pNqJo6ose/usr/share/perl
3256	/tmp/tmp.8pNqJo6ose/var/lib/dpkg/info
3500	/tmp/tmp.8pNqJo6ose/usr/share/zoneinfo
3508	/tmp/tmp.8pNqJo6ose/var/lib/dpkg
3580	/tmp/tmp.8pNqJo6ose/var/lib
4208	/tmp/tmp.8pNqJo6ose/bin
5012	/tmp/tmp.8pNqJo6ose/var
6536	/tmp/tmp.8pNqJo6ose/usr/lib/x86_64-linux-gnu/gconv
8944	/tmp/tmp.8pNqJo6ose/usr/bin
9376	/tmp/tmp.8pNqJo6ose/usr/share
9760	/tmp/tmp.8pNqJo6ose/lib/x86_64-linux-gnu
10116	/tmp/tmp.8pNqJo6ose/lib
14508	/tmp/tmp.8pNqJo6ose/usr/lib/x86_64-linux-gnu
17108	/tmp/tmp.8pNqJo6ose/usr/lib
37072	/tmp/tmp.8pNqJo6ose/usr
59876	/tmp/tmp.8pNqJo6ose
Built in /tmp/tmp.8pNqJo6ose
Image built at build/jessie.tar
Failed to load  correctly, expected id to be c37949263d998654631c5fdf2959838908384c1be7564fddf1510bed83047fc6, source in /tmp/tmp.zrR1tEhIza
```

cc/ @sameersbn @prydonius @james-w

**Comment by sameersbn** [comment]
@tuna can you check if you have `md5sum` and `sha256sum` in your PATH?

**Comment by ngtuna** [comment]
Yes I have it @sameersbn 

```
ubuntu@ip-10-0-0-192:~/minideb$ which md5sum
/usr/bin/md5sum
ubuntu@ip-10-0-0-192:~/minideb$ which sha256sum
/usr/bin/sha256sum
```

**Comment by james-w** [comment]
`Failed to load  correctly, expected id to be c37949263d998654631c5fdf2959838908384c1be7564fddf1510bed83047fc6, source in /tmp/tmp.zrR1tEhIza`

There's something missing after "Failed to load" which is

`docker load -i $TDIR/import.tar | awk '{print $4}'`

What do you get if you run

`docker load -i /tmp/tmp.zrR1tEhIza/import.tar`?

**Comment by ngtuna** [comment]
@james-w Nothing happens if I run that command.

**Comment by ngtuna** [comment]
Can you guys build it success from your end ?

**Comment by james-w** [comment]
Yes, I can still build locally.

I don't know why you would get no output from `docker load`. We can add a guard to give a better error, but it won't fix it for you.

What version of docker client/server are you using? I am using `17.04.0-ce` for both.

**Comment by ngtuna** [comment]
OK I am using a bit old docker engine. Will try with the new version and get back

**Comment by ngtuna** [comment]
OK So I ran `./buildall` on a fresh Ubuntu 16.04, docker `17.03.1-ce` (latest on xenial/stable amd64). These packages are required: `debootstrap`, `jq`, `debian-archive-keyring` and `dpkg-dev`.

And the build stopped at (I only paste the last lines where error happens).

```
================================
TEST: Check that all base packages are correctly installed, including dependencies
================================
Get:1 http://security.debian.org jessie/updates InRelease [63.1 kB]
Ign http://httpredir.debian.org jessie InRelease
Get:2 http://security.debian.org jessie/updates/main amd64 Packages [517 kB]
Get:3 http://httpredir.debian.org jessie Release.gpg [2373 B]
Get:4 http://httpredir.debian.org jessie Release [148 kB]
Get:5 http://httpredir.debian.org jessie/main amd64 Packages [9065 kB]
Fetched 9796 kB in 6s (1549 kB/s)
Reading package lists...

TEST: OK

================================
TEST: Check that install_packages doesn't loop forever on failures
================================
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package thispackagebetternotexist
apt failed, retrying
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package thispackagebetternotexist
apt failed, retrying

```

Not sure what `thispackagebetternotexist` means. I think this error doesn't relate to docker version.

cc/ @james-w @sameersbn

**Comment by ngtuna** [comment]
Oh... That's not an error. Dunno why the build stopped at that step but I just re-ran and it passed. Result at:

```
$ ls -la build
total 109316
drwxrwxr-x 2 ubuntu ubuntu     4096 May 31 07:18 .
drwxrwxr-x 5 ubuntu ubuntu     4096 May 31 07:06 ..
-rw-r--r-- 1 root   root      26519 May 31 07:13 jessie-repro.tar.log
-rw-r--r-- 1 root   root       8301 May 31 07:13 jessie-repro.tar.manifest
-rw-r--r-- 1 root   root   54917120 May 31 07:08 jessie.tar
-rw-rw-r-- 1 ubuntu ubuntu    26513 May 31 07:08 jessie.tar.log
-rw-r--r-- 1 root   root       8301 May 31 07:08 jessie.tar.manifest
-rw-r--r-- 1 root   root      24094 May 31 07:18 unstable-repro.tar.log
-rw-r--r-- 1 root   root       7872 May 31 07:18 unstable-repro.tar.manifest
-rw-r--r-- 1 root   root   56862720 May 31 07:15 unstable.tar
-rw-r--r-- 1 root   root      24088 May 31 07:15 unstable.tar.log
-rw-r--r-- 1 root   root       7872 May 31 07:15 unstable.tar.manifest
```

**Comment by james-w** [comment]
I think it was an error, a timeout on that test. I will try a PR to extend the timeout slightly and give a better error message if it is triggered.

**Comment by ngtuna** [comment]
Gotcha. Thanks for your support @james-w @sameersbn
